### ISSUE TITLE
Support source maps for generated code

### ISSUE THREAD

**Issue by benjamn** [facebook/regenerator#18]
The best place to implement source map support is probably in https://github.com/benjamn/recast, which is a dependency, but I wanted to have an issue here so that people know source maps are on the roadmap.

**Comment by appden** [comment]
It looks like recast supports source maps now, so is this still an issue?

**Comment by facebook-github-bot-8** [comment]
Thank you for reporting this issue and appreciate your patience. We've  notified the core team for an update on this issue. We're looking for a response within the next 30 days or the issue may be closed.

**Comment by matthiasg** [comment]
So is this still an issue ?

**Comment by jonnybonbon** [comment]
Yes i believe it is. My workaround is to not transpile generators when debugging locally as the latest version of chrome supports them anyway. In my case that meant removing the es2015 preset and installing and using its plugins as necessary to get a working system again (only "transform-es2015-modules-commonjs" for me but perhaps I'm behind the curve ;) )
